pi/index.html

```diff
@@ -1,212 +1,243 @@
-
-    <script src="../node_modules/webrtc-adapter/out/adapter.js"></script>
-    <script src="../node_modules/getusermedia-js/dist/getUserMedia.min.js"></script>
-    <script src="../node_modules/socket.io-client/dist/socket.io.js"></script>
-
-    <script src="../dist/infragram.js"></script>
-
-    <script type="x-shader/x-vertex" id="shader-vs"></script>
-    <script type="x-shader/x-fragment" id="shader-fs-template"></script>
-    <script type="x-shader/x-fragment" id="shader-fs"></script>
-
-  </head>
-
-  <body>
-  
-    <div class="container page" style="padding-top:10px;">
-
-       <div class="modal" tabindex="-1" role="dialog" id="preset-modal">
-         <div class="modal-dialog modal-lg" role="document">
-           <div class="modal-content">
-             <!--<div class="modal-header"></div>-->
-             <div class="modal-body">
-               <h3>Quick start</h3>
-               <p>Choose from these presets to get up and running quickly:</p>
-                 <table class="table">
-                   <tr>
-                     <td width="30%"><b>Raw</b></td>
-                     <td width="30%">Just what your camera sees.</td>
-                     <td><a class="btn btn-primary" id="preset_raw">Begin</a></td>
-                   </tr>
-                   <tr>
-                     <td><b>NDVI for <span style="color:blue;">BLUE</span> filters</b></td>
-                     <td>basic plant analysis for <a href="http://store.publiclab.org/product/infragram-diy-filter-pack">Infragram filter packs</a> and <a href="http://store.publiclab.org/product/infragram-webcam">webcams</a></td>
-                     <td><a class="btn btn-primary" id="preset_ndvi_blue">Basic</a>
-                         <a class="btn btn-success" id="preset_ndvi_blue_color">Colorized</a>
-                         <a class="btn btn-success" onClick="boost('blue')">Colorized + boosted (for Pi cameras)</a></td>
-                   </tr>
-                   <tr>
-                     <td><b>NDVI for <span style="color:red;">RED</span> filters</b></td>
-                     <td>basic plant analysis for <a href="http://store.publiclab.org/products/infragram-point-shoot-plant-cam">Infragram Point &amp; Shoot cameras</a></td>
-                     <td><a class="btn btn-primary" id="preset_ndvi_red">Basic</a>
-                         <a class="btn btn-success" id="preset_ndvi_red_color">Colorized</a>
-                         <a class="btn btn-success" onClick="boost('red')">Colorized + boosted (for Pi cameras)</a></td>
-                   </tr>
-                 </table>
-                 <p>Post a research note at <a href="http://publiclab.org">PublicLab.org</a> and email <a href="mailto:plots-dev@googlegroups.com">plots-dev@googlegroups.com</a> to suggest new presets.</p>
-
-	    </div>
-            <div class="modal-footer">
-              <a data-dismiss="modal" data-toggle="modal" class="btn">cancel</a>
-            </div>
```

Comment:
@Syed-Ansar Your linter/formatter has added more changes than necessary. You need to disable it and then undo these changes.